doc/source/getting_started/install.rst

```diff
@@ -139,6 +139,16 @@ pandas can be installed via pip from
 
     pip install pandas
 
+pandas can also be installed with sets of optional dependencies to enable certain functionality. For example,
+to install pandas with the optional dependencies to read Excel files.
+
+::
+
+    pip install pandas[excel]
```

Comment:
I think this won't work on mac,
```suggestion
    pip install "pandas[excel]"
```
should work everywhere

(likewise for the others)

https://github.com/psf/black/pull/3254#issuecomment-1236197864

Reply:
Good idea! Added